Presentation.Web/Helpers/CronStringHelper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Core.DomainModel.Advice;
+
+namespace Presentation.Web.Helpers
+{
+    public class CronStringHelper
+    {
+        // ┌───────────── minute (0 - 59)
+        // │ ┌───────────── hour (0 - 23)
+        // │ │ ┌───────────── day of the month (1 - 31)
+        // │ │ │ ┌───────────── month (1 - 12)
+        // │ │ │ │ ┌───────────── day of the week (0 - 6) (Sunday to Saturday;
+        // │ │ │ │ │                                   7 is also Sunday on some systems)
+        // │ │ │ │ │
+        // │ │ │ │ │
+        // * * * * * <command to execute>
+
+        public static string CronPerInterval(Scheduling interval, DateTime zeroTime)
+        {
+            switch (interval)
+            {
+                case Scheduling.Hour: return "0 * * * *"; 
+                case Scheduling.Day: return "0 8 * * *"; 
+                case Scheduling.Week: return $"0 8 * * {zeroTime.DayOfWeek:D}";
+                case Scheduling.Month: return $"0 8 {zeroTime.Day} * *";
+                case Scheduling.Quarter: return $"0 8 {zeroTime.Day} */3 *";
+                case Scheduling.Semiannual: return $"0 8 {zeroTime.Day} */6 *";
+                case Scheduling.Year: return $"0 8 {zeroTime.Day} */12 *";
```

Comment:
Bør denne ikke bare have måneden fra zeroTime på måneds pladsen?

Reply:
"*/3" kan ikke erstattes af f.eks. "7/3". Dette vil for cron betyde at f.eks. kvartals advis først startes i et nyt år i juli. Hvis man insisterer på at dette skal kunne lade sig gøre (istedet for den valgte løsning med kvartaler = jan, april, juli, oktober), så skal man oprette to cron jobs per advis og det giver ekstra kompleksitet ift redigering/oprydning.

Reply:
Ah .. det er "år" der er problemet - der skal der stå måneden .. jeg retter til